WiinUPro/Windows/MainWindow.xaml.cs

```diff
@@ -38,6 +40,17 @@ public MainWindow()
 
             WinBtStream.OverrideSharingMode = true;
             WinBtStream.OverridenFileShare = System.IO.FileShare.ReadWrite;
+            trayIcon = new System.Windows.Forms.NotifyIcon();
+            trayIcon.Icon = new System.Drawing.Icon(@"..\..\app.ico");
+            trayIcon.Visible = false;
+            trayIcon.DoubleClick +=
+                delegate (object sender, EventArgs args)
+                {
+                    this.Show();
+                    trayIcon.Visible = false;
+                };
+            this.trayIcon.ContextMenuStrip = new System.Windows.Forms.ContextMenuStrip();
+            this.trayIcon.ContextMenuStrip.Items.Add("Exit", null, this.ContextMenu_TrayExit);

```

Comment:
At the moment the context menu only contains an option to completely close the application. Additional menu items could be added, but I couldn’t think of any other items which would be particularly useful to access from here.